### PR TITLE
changed to expression body #hacktoberfest

### PR DESCRIPTION
**Pull request by logeshpalani98**
### Bugs Fixed ###

> Code format changed

- Fixes #

### API Changes ###

<!-- List all API changes here (or just put None), example:

Added: 
 
- `string Class.Property { get; set; }`
- `void Class.Method();`

Changed:

 - `object Cell.OldPropertyName` => `object Cell.NewPropertyName`
 
If there is an entirely new API, then you can use a more verbose style:

```csharp
public static class NewClass {
    public static int SomeProperty { get; set; }
    public static void SomeMethod(string value);
}
``` -->

### Behavioral Changes ###

<!-- Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase. -->

### PR Checklist ###
<!-- Please check all the things you did here and double-check that you got it all, or state why you didn't do something -->

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard
<!-- If at all possible, please update/add the documentation on the repo below. We would very much appreciate that. If you are unable to, please consider at least opening an issue on the repo below so we know that Docs still need to be adjusted/created. Thanks! <3 -->
- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

File: XamarinCommunityToolkit/Views/Snackbar/Helpers/iOS/IOSSnackBar.ios.cs [hunk 1]
```diff
@@ -97,12 +97,9 @@ public IOSSnackBar Show()
 
 		BaseSnackbarView GetSnackbarView()
 		{
-			if (Action != null && !string.IsNullOrEmpty(ActionButtonText))
-			{
-				return new ActionMessageSnackbarView(this);
-			}
-
-			return new MessageSnackbarView(this);
+			return Action != null && !string.IsNullOrEmpty(ActionButtonText)
```
Comment (VladislavAntonyuk) [inline XamarinCommunityToolkit/Views/Snackbar/Helpers/iOS/IOSSnackBar.ios.cs:100]: this changes already done in SnackBarOptions PR.

File: XamarinCommunityToolkit/Views/Snackbar/Helpers/macOS/SnackbarViews/MessageSnackbarView.macos.cs [hunk 2]
```diff
@@ -4,8 +4,7 @@ namespace Xamarin.CommunityToolkit.UI.Views.Helpers.macOS.SnackbarViews
 {
 	class MessageSnackbarView : BaseSnackbarView
 	{
-		public MessageSnackbarView(MacOSSnackBar snackbar)
-			: base(snackbar)
+		public MessageSnackbarView(MacOSSnackBar snackbar) : base(snackbar)
```
Comment (pictos) [inline XamarinCommunityToolkit/Views/Snackbar/Helpers/macOS/SnackbarViews/MessageSnackbarView.macos.cs:7]: can you please revert this change?

**Comment by jfversluis** [comment]
Hi @logeshpalani98 I appreciate you making your first steps with us and contributing to OSS to thank you for that.

I don't think this provides enough value though to count towards the Hacktoberfest challenge. Also, it seems parts of this change are already part of another open PR. I'm closing this one, I hope this doesn't discourage you for making future contributions.

Please check out the contribution documentation on the repo on how to go about it. The very least would be to work of an open issue.

Thanks!
